docs/upgrade/automatic.md

```diff
@@ -34,6 +34,7 @@ The following table outlines the supported upgrade paths.
 
 | Installed Version | Supported Upgrade Versions |
 | --- | --- |
+| v1.6.0 | [v1.6.1](./v1-6-0-to-v1-6-1.md) |
```

Comment:
Shouldn't there also be an upgrade path `v1.5.0/v1.5.1/v1.5.2 --> v1.6.1`?

Reply:
YES. @jillian-maroket, please help correct this.